contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.8.0--2.9.0.sql

```diff
@@ -350,6 +350,5087 @@ END;
 $$
 LANGUAGE plpgsql STABLE;
 
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    ALTER FUNCTION sys.babelfish_conv_helper_to_time(TEXT, BOOL, NUMERIC) RENAME TO babelfish_conv_helper_to_time_deprecated_in_2_9_0_1;
+EXCEPTION WHEN OTHERS THEN
```

Comment:
Can we be more specific about which error code to catch instead of silently ignoring all kinds of errors?

Reply:
Done